Code/GraphMol/ChemReactions/ReactionRunner.cpp

```diff
@@ -1068,7 +1070,7 @@ std::vector<MOL_SPTR_VECT> run_Reactant(const ChemicalReaction &rxn,
   }
 
   if (!ReactionRunnerUtils::getReactantMatches(
-          reactants, rxn, matchesByReactant, reactantIdx)) {
+          reactants, rxn, matchesByReactant, 1000, reactantIdx)) {
```

Comment:
Why is this hard coded and not related to maxProducts?

I'm forgetting the logic here.

Reply:
`runReactant()` doesn't have a maxProducts argument
The hard coding to 1000 is just because we have to have something here. That happens to be the default for the `maxMatches` in the substructure search.

Reply:
Ah, this is runReactant not runReactants...

I'm almost 100% sure that this function doesn't really need a maxProduct arg... but if we want to make it consistent with the current bug, we can add it.

Reply:
I’d rather not, that’s an enhancement (of sorts) since it changes the API